packages/cli/src/utils/build.js

```diff
@@ -290,6 +290,19 @@ const maybeNotifyAboutOutdated = () => {
   }
 };
 
+const maybeRunBuildScript = async (options = {}) => {
+  const npmScripts = _.get(
+    require(path.resolve(options.cwd || '.', './package.json')),
+    'scripts'
+  );
+
+  if ('zapier-build' in npmScripts) {
```

Comment:
We need to make sure `npmScripts` is not undefined here, otherwise we'll get a `TypeError` if `scripts` isn't defined in `package.json`.

Suggested change:

```js
if (npmScripts && 'zapier_build' in npmScripts) {
  ...
}
```